plugins/Toolbox/resources/qml/ToolboxProgressButton.qml

```diff
@@ -25,43 +25,36 @@ Item
 
     width: UM.Theme.getSize("toolbox_action_button").width
     height: UM.Theme.getSize("toolbox_action_button").height
-
-    Cura.PrimaryButton
+    fixedWidthMode: true
+    text:
```

Comment:
Might want to have a look at states for this (https://doc.qt.io/qt-5/qml-qtquick-state.html) I find that a bit cleaner than very long if / else if statements. It also nicely groups the behavior on state change.

Reply:
I didn't do this change. Maybe next time. But seems like a good idea.